data/seeds/003-companies.js

```diff
@@ -0,0 +1,20 @@
+exports.seed = function(knex) {
+  return knex("companies")
+    .del()
+    .then(function() {
+      return knex("companies").insert([
+        {
+          company_name: "Lambda School",
+          industry: "education",
+          password: "test1234",
+          admins_id: 1
```

Comment:
This should be admin_id instead of admins_id... It's referring to a single admin

Reply:
Solved this myself.